### DISCUSSION TITLE
Switch to KPN

### DISCUSSION THREAD

**Discussion by xHuubx** [fabianishere/udm-iptv#22] [Q&A] [answered]
I have successfully completed the steps and am now running it with my current provider XS4ALL. Because of the news in the media that KPN is going to stop with XS4ALL, I'm thinking about switching to KPN. Do I just have to repeat the steps as described or are there other things I have to take into account?

**Comment by fabianishere** [comment, marked as answer]
I expect the process to be the same for KPN.
